experimental/client-app/package.json

```diff
@@ -13,9 +13,7 @@
     "start:build": " web-dev-server --root-dir dist --app-index index.html --open",
     "analyze": "cem analyze --litelement",
     "lint:css": "stylelint \"src/**/*.js\"",
-    "server": "node server/proxy.js",
-    "start": "THEME=cymbal NODE_ENV=development web-dev-server",
-    "dev": "run-p server start"
+    "start": "THEME=cymbal NODE_ENV=development web-dev-server"
```

Comment:
What does `NODE_ENV=development` do for this project?

Reply:
This is to describe if the app is in dev, staging or prod environment and bundle / minify as directed in rollup. Not used actively now, so I can remove to prevent confusion.